webui/react/src/components/Transfer.tsx

```diff
@@ -78,7 +78,25 @@ const Transfer: React.FC<Props> = ({
 
   const renderEntry = useCallback(
     (entryName: string) => {
-      const sentenceEntryName = entryName === 'id' ? 'ID' : camelCaseToSentence(entryName);
+      const renameEntry = (): string => {
```

Comment:
this isnt ideal but its component is only used for experiment table, so it should be fine.
we should refactor around experiment table when we switch to the new table lib